### PR TITLE
fix: allow '.default' to be ignored during schema validation

### LINKED ISSUE #20388: DTS Import is failing on schema detection
### Required System information

- Environment: development
- OS: darwin-arm64
- Strapi Version: 5.0.0-beta.8
- Node/Yarn Version: npm/10.2.4 node/v20.11.1 darwin arm64 workspaces/false
- Edition: Community
- Database: sqlite
- Is your project Javascript or Typescript:

### Describe the bug

DTS Import  fails with warning messages after completing no files were imported.

### Steps to reproduce the behavior

See video

https://github.com/strapi/strapi/assets/6153188/254b49dd-d3af-4aeb-af43-4c489df1415b

### Expected behavior

Should import data.

### Screenshots

No data imported

### Code snippets

If applicable, add code samples to help explain your problem.

### PR DESCRIPTION
**Pull request by Convly**
### What does it do?

- Add 'default' to the list of ignorable attributes's properties
- Sanitize schemas to be valid JSON to make sure no invalid JSON properties are present during the checks

### Why is it needed?

strapi import on identical apps is currently broken because of 'default' not being present all the time

### How to test it?

See #20388 for reproduction

### Related issue(s)/PR(s)

fix #20388

### CODE REVIEW

**Comment by Convly** [comment]
This is the first simple fix for this issue.

However, it'll still break transfers using the 'exact' strategy.

`default` is present in the diff because source schemas (extracted from the archive) are stringified then parsed again, whereas the destination ones (taken from the running Strapi instance) still contain invalid JSON properties such as functions (e.g. `default`).

One fix could be to force the programmatic `getSchemas` to return valid JSON data structures (manually excluding the functions & other invalid values)

What do you think? @innerdvations @Bassel17

**Comment by innerdvations** [comment]
Yeah, I agree that this works as a quick fix for now, and then the "real" fix should be that `getSchemas` should use the exact same logic as the export method to ensure we don't have any more incongruencies like this.

File: packages/core/data-transfer/src/engine/validation/schemas/index.ts [hunk 1]
```diff
@@ -13,21 +13,25 @@ const isAttributeIgnorable = (diff: Diff) => {
     // Need a valid string attribute name
     typeof diff.path[1] === 'string' &&
     // The diff must be on ignorable attribute properties
-    ['private', 'required', 'configurable'].includes(diff.path[2])
+    ['private', 'required', 'configurable', 'default'].includes(diff.path[2])
```
[the remark this answers is not in the document]
Comment (Convly) [reply packages/core/data-transfer/src/engine/validation/schemas/index.ts:16]: Basically, we shouldn't have a difference on this field every time (hence the second fix), but if for some reason it happens anyway (user modified their schema), then we should probably ignore it anyway

**Review by Bassel17 — APPROVED** [review]
LGTM

**Pull request merged**